src/legacy/core_plugins/kibana/public/kibana.js

```diff
@@ -42,6 +42,7 @@ import 'uiExports/shareContextMenuExtensions';
 import 'uiExports/interpreter';
 
 import 'ui/autoload/all';
+import 'ui/error_url_overflow';
```

Comment:
Are we sure these 3 imports are enough and that the feature isn't used elsewhere?